app/services/pii/attribute.rb

```diff
@@ -0,0 +1,25 @@
+module Pii
+  class Attribute
+    attr_accessor :user, :norm
+
+    def initialize(user: nil, norm: nil)
```

Comment:
`user` means user-provided? Should we just call it "raw" or "original" maybe? calling it `user` makes me think it should be our activerecord model.